Cython/Compiler/Main.py

```diff
@@ -178,7 +178,7 @@ def _check_pxd_filename(pos, pxd_pathname, qualified_name):
                     " Please use the normal Python package directory layout." % pxd_filename, level=1)
 
     def find_module(self, module_name, relative_to=None, pos=None, need_pxd=1,
-                    absolute_fallback=True):
+                    absolute_fallback=True, relative_import=False):
```

Comment:
I think that this function deserves some refactoring. At least I find confusing `relative_to` parameter which has nothing with relative imports but contains just the `from` part of the import. I am not sure what could be the better name...

Reply:
Maybe `from_module` or something similar.

Reply:
I wonder if it could at least be split in two - there seems to be a part that finds the module and a part that loads the pxd, and it maybe doesn't help to have them in one long function

Reply:
I found out that splitting `find_module()` into several separate functions is rather non-trivial. Hence, I am not going to do that in this PR to keep it light. But still I have renamed `relative_to` to better name.